stable/logstash/Chart.yaml

```diff
@@ -3,7 +3,7 @@ description: Logstash is an open source, server-side data processing pipeline
 icon: https://www.elastic.co/assets/blt86e4472872eed314/logo-elastic-logstash-lt.svg
 home: https://www.elastic.co/products/logstash
 name: logstash
-version: 2.1.0
+version: 2.1.1
```

Comment:
Thanks for your PR, this PR included the new feature. I believe that may have an upgrade on the MINOR version, e.g 2.2.0

Reply:
Thanks, I pushed an update accordingly.